Bottom: cfc1c1f9ce7f7a1b1317213b667bc137564dfebb
Top:    518b870dc0f789bcdb316238639ed0179f22c559
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 19:16:15 +0000

Remove left over debug


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 7039457..55571cf 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1251,8 +1251,6 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
 #endif
                                 copper_appearance,
                                 NULL));
-
-      //break;
     }
